<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XOLO Project Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="topMargin">
          <number>2</number>
         </property>
         <item>
          <widget class="QTabWidget" name="tabWidget">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="projects_tab">
            <attribute name="title">
             <string>Projects</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <property name="bottomMargin">
                <number>20</number>
               </property>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <property name="rightMargin">
                  <number>10</number>
                 </property>
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_4">
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_2">
                     <property name="topMargin">
                      <number>10</number>
                     </property>
                     <property name="bottomMargin">
                      <number>10</number>
                     </property>
                     <item>
                      <widget class="QLabel" name="label">
                       <property name="text">
                        <string>Name:</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLineEdit" name="project_name_lineEdit"/>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_3">
                     <property name="topMargin">
                      <number>10</number>
                     </property>
                     <property name="bottomMargin">
                      <number>10</number>
                     </property>
                     <item>
                      <widget class="QLabel" name="label_2">
                       <property name="text">
                        <string>fps:</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLineEdit" name="fps_lineEdit"/>
                     </item>
                     <item>
                      <widget class="QLabel" name="label_3">
                       <property name="text">
                        <string>resolution:</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLineEdit" name="ress_lineEdit"/>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <widget class="Line" name="line_2">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_4">
                     <item>
                      <widget class="QLabel" name="label_4">
                       <property name="text">
                        <string>Delete Project:</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QComboBox" name="projects_delete_comboBox"/>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <widget class="QPushButton" name="delete_project_pushButton">
                     <property name="styleSheet">
                      <string notr="true">background-color: rgb(192, 28, 40);</string>
                     </property>
                     <property name="text">
                      <string>Delete</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <widget class="Line" name="line_4">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                  </widget>
                 </item>
                 <item alignment="Qt::AlignHCenter|Qt::AlignTop">
                  <widget class="QPushButton" name="create_project_pushButton">
                   <property name="styleSheet">
                    <string notr="true">background-color: rgb(51, 209, 122);</string>
                   </property>
                   <property name="text">
                    <string>Create</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="Line" name="line">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QGroupBox" name="groupBox">
                 <property name="title">
                  <string>Projects info</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_6">
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_5">
                    <item>
                     <widget class="QTableWidget" name="projects_tableWidget">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="frameShape">
                       <enum>QFrame::StyledPanel</enum>
                      </property>
                      <property name="sizeAdjustPolicy">
                       <enum>QAbstractScrollArea::AdjustToContents</enum>
                      </property>
                      <attribute name="horizontalHeaderCascadingSectionResizes">
                       <bool>false</bool>
                      </attribute>
                      <column>
                       <property name="text">
                        <string>Name</string>
                       </property>
                      </column>
                      <column>
                       <property name="text">
                        <string>Type</string>
                       </property>
                      </column>
                      <column>
                       <property name="text">
                        <string>Status</string>
                       </property>
                      </column>
                     </widget>
                    </item>
                    <item>
                     <widget class="Line" name="line_3">
                      <property name="orientation">
                       <enum>Qt::Vertical</enum>
                      </property>
                     </widget>
                    </item>
                    <item alignment="Qt::AlignTop">
                     <widget class="QPushButton" name="update_projects_pushButton">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="minimumSize">
                       <size>
                        <width>0</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="styleSheet">
                       <string notr="true">background-color: rgb(28, 113, 216);</string>
                      </property>
                      <property name="text">
                       <string>Update</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="shots_tab">
            <attribute name="title">
             <string>Shots</string>
            </attribute>
           </widget>
           <widget class="QWidget" name="assets_tab">
            <attribute name="title">
             <string>Assets</string>
            </attribute>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>684</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
